Content.Shared/Movement/Systems/SharedJetpackSystem.cs

```diff
@@ -117,21 +116,22 @@ private void OnJetpackToggle(EntityUid uid, JetpackComponent component, ToggleJe
         if (args.Handled)
             return;
 
-        if (TryComp<TransformComponent>(uid, out var xform) && !CanEnableOnGrid(xform.GridUid))
+        if (!CanEnable(Transform(uid).Coordinates))
         {
             if (_timing.IsFirstTimePredicted)
-                _popups.PopupEntity(Loc.GetString("jetpack-no-station"), uid, args.Performer);
+                _popups.PopupEntity(Loc.GetString("jetpack-under-gravity"), uid, args.Performer);
```

Comment:
See below.

Also the localisation change seems redundant.

Reply:
localisation wasn't clear enough (imo) so I changed it